tests/core/creators/manual/detached.html

```diff
@@ -0,0 +1,39 @@
+<button id="attachButton">Attach editor parent</button>
+
+<div>
+	<div id="editor1">
+		<p>Lorem ipsum dolor sit amet 1.</p>
+	</div>
+</div>
+
+
+<div id="fakedId1"></div>
+
+
+<script>
+	var editorElement = document.getElementById("editor1");
+
+	var textArea = document.createElement("textarea");
+
+	editorElement.appendChild(textArea);
+
+	editorElement.parentNode.removeChild(editorElement);
```

Comment:
I'm not sure why you are using `textarea` and attaching editor as it's child :thinking: AFAIR `textarea` couldn't have other elements inside (well, they will be treated as text).

Maybe, simply detaching parent `div` (or editor element itself) would be enough?

Reply:
Actually, I'm not attaching editor as a child of `textarea`. I add `textarea` as a child of editor `div`.  
  
Originally I tried to recreate the same structure as we can observe in angular, but you are right: 

> Maybe, simply detaching parent `div` (or editor element itself) would be enough?

It should be enough. I changed this.